file-system-access/WEB_FEATURES.yml

```diff
@@ -1,3 +1,7 @@
 features:
 - name: file-system-access
   files: "**"
+- name: origin-private-file-system
```

Comment:
`file-system-access` will also apply to these files as well. Is that intended?

Reply:
I left it alone in this PR, but it's not good. The tests for file-system-access end up being the same as OPFS plus an idlharness test which is shallow, and covers OPFS a bit too.

I think we should probably unmap these tests, but wanted to bring it up with Kadir first, I made a note of it.